### ISSUE TITLE
After installation: "The addon 'ember-cli-htmlbars' requires an Ember CLI version of 0.1.2 or above, but you are running null."

### ISSUE THREAD

**Issue by sheriffderek** [crodriguez1a/ember-themed-syntax#3]
After installation: Server breaks and gives the error: "The addon 'ember-cli-htmlbars' requires an Ember CLI version of 0.1.2 or above, but you are running null."

I was trying to use this addon and a few others, so I wasn't sure it was a CLI issue or ember-autoserve. I spun up a bunch of apps with "ember-cli": "2.7.0-beta.2", - and tried every order and combination of install with those addons. After all that - it's seems to be that this is the addon that brings it up. I installed 5 or so different addons, and only after this particular addon is installed does the server break and return this error.

I'm not sure how that could be. To recreate, all you need to do is generate a fresh app with the latest CLI and install this addon.

This is the package.json I ended up with:

```
"autoprefixer": "6.3.6",
"broccoli-asset-rev": "^2.4.2",
"ember-ajax": "^2.0.1",
"ember-autoserve": "0.0.4",
"ember-cli": "2.7.0-beta.2",
"ember-cli-app-version": "^1.0.0",
"ember-cli-babel": "^5.1.6",
"ember-cli-dependency-checker": "^1.2.0",
"ember-cli-htmlbars": "^1.0.3",
"ember-cli-htmlbars-inline-precompile": "^0.3.1",
"ember-cli-inject-live-reload": "^1.4.0",
"ember-cli-jshint": "^1.0.0",
"ember-cli-qunit": "^2.0.0",
"ember-cli-release": "0.2.8",
"ember-cli-sri": "^2.1.0",
"ember-cli-stylus": "1.0.6",
"ember-cli-test-loader": "^1.1.0",
"ember-cli-uglify": "^1.2.0",
"ember-export-application-global": "^1.0.5",
"ember-load-initializers": "^0.5.1",
"ember-resize": "0.0.14",
"ember-resolver": "^2.0.3",
"ember-themed-syntax": "0.0.4",
"loader.js": "^4.0.1"
```

**Comment by crodriguez1a** [comment]
@sheriffderek Thanks for describing the issue in such detail. I'll take a look at it and get back.

**Comment by crodriguez1a** [comment]
@sheriffderek Okay, seems like it could be related to this issue. https://github.com/ember-cli/ember-cli/issues/5973. I'm looking into it.

**Comment by crodriguez1a** [comment]
@sheriffderek This issue should now be fixed with @0.0.5. Please test and let me know if your issue persists. Thx.

**Comment by sheriffderek** [comment]
Seems OK now! Thanks for checking into it.
